tests/network/probes.go

```diff
@@ -82,8 +82,8 @@ var _ = SIGDescribe("[ref_id:1182]Probes", func() {
 				Expect(tests.PodReady(tests.GetRunningPodByVirtualMachineInstance(vmi, util.NamespaceTestDefault))).To(Equal(condition))
 			}
 
+			libnet.SkipWhenClusterNotSupportIpFamily(virtClient, IPFamily)
```

Comment:
`skipIfNotSupportedCluster` is an internal method of expose.go that checks whether the ipfamily of a service is supported in the current cluster.
The method is not relevant to probes.go since it doesn't use a service.